parser.y

```diff
@@ -7374,6 +7467,26 @@ HashString:
 		$$ = $1
 	}
 
+RoleSpec:
+	Rolename
+	{
+		roleSpec := &ast.UserSpec{
+			User: $1.(*auth.UserIdentity),
+		}
+		roleSpec.AuthOpt = nil
```

Comment:
Is there a password for a role? and what is it?

Reply:
Role doesn't have password, so set this field to `nil`.